pset-iterators.js

```diff
@@ -92,7 +107,11 @@ Tip: your new array should contain 45 elements
 console.log('Problem 3:')
 
 // Add your code below this line
+const presidentialParties = presidents.map((party) => {
+  return party.party
+})
 
+console.log(presidentialParties)
```

Comment:
@jct117 Good job on this solution here, to use the map method to return all of the president's party.

Take a look at an alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L120)